cmake/AddRCInfo.cmake

```diff
@@ -30,6 +30,13 @@ if ("${git_branch}" STREQUAL "master")
 else()
   set(build_release ON)
 endif()
+set(original_filename "rime.dll")
+if(MSVC)
+  set(CMAKE_RC_FLAGS "${CMAKE_RC_FLAGS} -DMSVC")
```

Comment:
nit: `MSVC` gets me confused until I saw it used as the conditions in the RC file.

for better readability, I suggest naming it `LOCALIZE_VERSION_INFO`.